source/renderer/webpack.config.js

```diff
@@ -90,6 +90,7 @@ module.exports = {
       'process.env.MOBX_DEV_TOOLS': process.env.MOBX_DEV_TOOLS || 0,
       'process.env.BUILD_NUMBER': JSON.stringify(process.env.BUILD_NUMBER || 'dev'),
       'process.env.REPORT_URL': JSON.stringify(reportUrl),
+      'process.env.WALLET_PORT': JSON.stringify(process.env.WALLET_PORT || ''),
```

Comment:
@rvl here we'll need fallback port `8090` instead of empty string to keep Daedalus working in development env...

Reply:
environment.js has a default, and i think it will trigger when this part sets it to `''`?

Reply:
That is a valid point. Let's keep it as is...